docs/access/linux.rst

```diff
@@ -16,7 +16,9 @@ Starting Kolibri on Linux will differ depending on the method you used to instal
     .. code-block:: bash
 
       kolibri start
-      
+
+    .. note::
+      By default **Kolibri** starts on 8080 port. If the default port is in use, then refer to `this <https://kolibri.readthedocs.io/en/latest/manage/options_ini.html#run-kolibri-from-a-different-port>`_ doc for running it from a different port.
```

Comment:
Just a heads-up that the link should use use an [internal target](https://docutils.sourceforge.io/docs/user/rst/quickref.html#hyperlink-targets) because otherwise the link will break when the path prefix and domain change.

Reply:
Yes, thanks for reviewing. I've updated by adding the cross reference to the doc. Please check now and let me know if any other change is required.

Reply:
@indirectlylit please check